MAVProxy/modules/mavproxy_speech.py

```diff
@@ -1,26 +1,209 @@
 #!/usr/bin/env python
-'''tune command handling'''
+'''
 
-import time, os
+Speech Engine for audio announcements of MAVLink messages during flight.
+
+Suggested usage: Add initialization commands to .mavinit.scr
+
+Example commands:
+    module load speech
+    speech add VFR_HUD.airspeed onchange every=3.0 as=airspeed format=%0.0f threshold=2.0
```

Comment:
Why not dispense with `as` and require the user to put that in the format?

You could default format to "airspeed %0.0f" (replacing airspeed with whatever is after the "." in the nominated field...

Allowing `VFR_HUD.airspeed` to be a Python expression would be nice; we already have code floating around which lets you pull out field-y looking things from expressions.